File: college/php/factorial/factorial.php
<html>
<body>

<h2 align=center> factorial </h2>
<form action="" method="post">
enter the number: <input type="text" name="num"></br>
<button type="submit"> Calcuate </button>
</form>


<?php

if($_POST) {
  $number = $_POST['num'];
  $fac = 1;

  while($number > 1) {
    $fac *= $number;
    $number--;
  }

  echo "factorial is : $fac";
}

?>


</body>
</html>

File: diploma/S4/psp/C3/factorial.c
#include <stdio.h>

int main() {

  int n;
  int fac = 1;

  printf("enter the number: ");
  scanf("%d", &n);

  for(int i = 1; i <= n; i++) {
    fac = fac * i; 
  }

  printf("the factorial of %d is %d\n", n, fac);
}
